### ISSUE TITLE
Crash on long press a floating action button

### ISSUE THREAD

**Issue by Jhonnyc** [flutter/flutter#17938]
## Steps to Reproduce

Crash on long press a floating action button, release (somewhere else on the screen) and long press
again.

1. Fresh install, create an app
2. Create an app using 'flutter create test_app'
3. Run on 'iPhone X' simulator
4. Long press on floatingActionButton, then move cursor to other part of the screen and release
5. Second Long press on floatingActionButton
6. KABOOM - app crashes

## Logs

Launching lib/main.dart on iPhone X in debug mode...
Starting Xcode build...
Xcode build done.
Syncing files to device iPhone X...
*** First throw call stack:
(
	0   CoreFoundation                      0x000000010f6411e6 __exceptionPreprocess + 294
	1   libobjc.A.dylib                     0x000000010d21a031 objc_exception_throw + 48
	2   CoreFoundation                      0x000000010f646472 +[NSException raise:format:arguments:] + 98
	3   Foundation                          0x000000010ccf364f -[NSAssertionHandler handleFailureInFunction:file:lineNumber:description:] + 165
	4   Flutter                             0x000000010de53349 _ZN5shell19AccessibilityBridge11HandleEventEP12NSDictionaryIP8NSStringP11objc_objectE + 269
	5   Flutter                             0x000000010de3f8bc __48-[FlutterBasicMessageChannel setMessageHandler:]_block_invoke + 124
Lost connection to device.
	6   Flutter                             0x000000010de561c6 _ZNK5shell21PlatformMessageRouter21HandlePlatformMessageEN3fxl6RefPtrIN5blink15PlatformMessageEEE + 166<…>

Yonis-MacBook-Pro:Flutter yoni$ flutter analyze
Analyzing Flutter...                                             
No issues found! (ran in 1.2s)

```
```

**Comment by jonahwilliams** [comment]
What is the output of `flutter doctor`?

Looks similar to the issue fixed with https://github.com/flutter/engine/pull/5349.  Should be available on Flutter master channel the next time there is an engine roll (soon)

**Comment by Jhonnyc** [comment]
Running 'flutter doctor' gives me the following - 

Doctor summary (to see all details, run flutter doctor -v):
[✓] Flutter (Channel beta, v0.4.4, on Mac OS X 10.13.3 17D102, locale en-CA)
[!] Android toolchain - develop for Android devices (Android SDK 27.0.0)
    ! Some Android licenses not accepted.  To resolve this, run: flutter doctor --android-licenses
[✓] iOS toolchain - develop for iOS devices (Xcode 9.3.1)
[✓] Android Studio
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
[✓] Android Studio (version 3.0)
    ✗ Flutter plugin not installed; this adds Flutter specific functionality.
    ✗ Dart plugin not installed; this adds Dart specific functionality.
[✓] IntelliJ IDEA Community Edition (version 2018.1.4)
[✓] Connected devices (1 available)

! Doctor found issues in 1 category.


I'm using IntelliJ as an IDE so the android studio output is not relevant.
Also - I accepted the license but it kinda repeat itself even after going through the process twice (I found it as a resolved bug already, but the solution did not work for me).
